Tasks/Maven/maventask.ts

```diff
@@ -155,21 +201,24 @@ async function execBuild() {
 
             // 2. Apply any goals for static code analysis tools selected by the user.
             mvnRun = sqMaven.applySonarQubeArgs(mvnRun, execFileJacoco);
-             mvnRun = codeAnalysisOrchestrator.configureBuild(mvnRun);
+            mvnRun = codeAnalysisOrchestrator.configureBuild(mvnRun);
 
             // Read Maven standard output
             mvnRun.on('stdout', function (data) {
                 processMavenOutput(data);
             });
 
             // 3. Run Maven. Compilation or test errors will cause this to fail.
-            return mvnRun.exec(); // Run Maven with the user specified goals
+            return mvnRun.execSync(util.getExecOptions());
```

Comment:
execSync() will block right? We should use execSync() for very quick operations only.